Presentation/Sources/Presentation/View/MainView.swift

```diff
@@ -0,0 +1,230 @@
+    static let labelFontSize: CGFloat = 50
+    static let numberOfLines: Int = 0
+    static let titleTop: CGFloat = 30
+    static let titleLeading: CGFloat = 0
+    static let titleTrailing: CGFloat = 0
+    static let griffindorImage: String = "griffindor_image"
+    static let slytherinImage: String = "slytherin_image"
+    static let hufflepuffImage: String = "hufflepuff_image"
+    static let ravenclawImage: String = "ravenclaw_image"
+    static let spellsTitle: String = "Spells"
+    static let wizardsTitle: String = "Wizards"
+    static let fontName: String = "Palatino-BoldItalic"
+    static let buttonFontSize: CGFloat = 30
+    static let shadowRadius: CGFloat = 1
+    static let shadowOpacity: Float = 1.0
+    static let shadowOffset: Int = 3
+    static let cornerRadius: CGFloat = 20
+    static let zeroConstraint: CGFloat = 0
+    static let imageConstantDimen: CGFloat = 190
+    static let shiftLeft: CGFloat = -100
+    static let shiftRight: CGFloat = 100
+    static let upImageTop: CGFloat = 100
+    static let downImageTop: CGFloat = 10
+    static let buttonTop: CGFloat = 120
+    static let buttonConstantHeight: CGFloat = 60
+    static let buttonConstantWidth: CGFloat = 140
+}
+
+// MARK: - Class
+
+class MainView: UIView {
+
+    // MARK: - Propeties
+
+    let backgroundImageView: UIImageView = {
+        let imageView = UIImageView()
+        imageView.image = UIImage(named: Constants.background)
+        return imageView
+    }()
+    let titleLabel: UILabel = {
+        let label = UILabel()
+        label.font = UIFont(name: Constants.fontName, size: Constants.labelFontSize)
+        label.text = Constants.labelTitle
+        label.lineBreakMode = .byWordWrapping
+        label.numberOfLines = Constants.numberOfLines
+        label.textColor = .black
+        label.textAlignment = .center
+        label.layer.shadowColor = Color.lightGray.cgColor
+        label.layer.shadowRadius = Constants.shadowRadius
+        label.layer.shadowOpacity = Constants.shadowOpacity
+        label.layer.shadowOffset = CGSize(width: Constants.shadowOffset, height: Constants.shadowOffset)
+        label.layer.masksToBounds = false
+        return label
+    }()
+    let griffindorImageView: UIImageView = {
+        let imageView = UIImageView()
+        imageView.image = UIImage(named: Constants.griffindorImage)
+        return imageView
+    }()
+    let slytherinImageView: UIImageView = {
+        let imageView = UIImageView()
+        imageView.image = UIImage(named: Constants.slytherinImage)
+        return imageView
+    }()
+    let hufflepuffImageView: UIImageView = {
+        let imageView = UIImageView()
+        imageView.image = UIImage(named: Constants.hufflepuffImage)
+        return imageView
+    }()
+    let ravenclawImageView: UIImageView = {
+        let imageView = UIImageView()
+        imageView.image = UIImage(named: Constants.ravenclawImage)
+        return imageView
+    }()
```

Comment:
The images have to be tapeable, it's on the description of the exercise.

Reply:
Got it, I thought that now it was going without the tap, but I put it there.